css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 73.5%;
+  top: 5%;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 51%;
```

Comment:
Instead of pushing the text down from the top, why not stick it to the bottom?

Reply:
WHOOPS DIDN'T SEE THIS SUGGESTION. Will work it now.

Reply:
I feel like a goober now. Corrections made!